src/js/navitron.js

```diff
@@ -135,7 +145,13 @@
                             $pane.find(selectors.CONTENT).attr('tabindex', 0).focus();
 
                             plugin._trigger('onShown', { pane: $pane });
-                        });
+                        };
+
+                        if (typeof window.requestAnimationFrame === 'function') {
```

Comment:
I wonder if it's useful to refactor this all a bit --- rather than repeated `if...else` statements, we define a function that accepts a callback and passes it to rAF if available, or calls it directly if not?